src/Models/NonhydrostaticModels/boundary_mass_fluxes.jl

```diff
@@ -79,7 +79,7 @@ initialize_boundary_mass_flux(velocity, ::Nothing, side) = NamedTuple()
 initialize_boundary_mass_flux(velocity, bc, side) = NamedTuple()
 
 needs_mass_flux_correction(::IOBC) = false
-needs_mass_flux_correction(bc::OBC) = bc.classification.matching_scheme !== nothing
+needs_mass_flux_correction(bc::OBC) = bc.classification.scheme !== nothing
```

Comment:
```suggestion
needs_mass_flux_correction(bc::OBC) = true
```

Reply:
I don't think that'd work because that would also flag OpenBoundaryConditions without a matching scheme as needing correction, where we're basically imposing a flow. I think we'd like those boundary conditions to have the exact flow/flux we impose on them, no?

Reply:
Isn't that took care of here?
```julia
needs_mass_flux_correction(::IOBC) = false
```

Reply:
Ah, that's right! I missed that. This PR has been sitting here so long I forgot the details lol